examples/FMP_simple_temp.jl

```diff
@@ -0,0 +1,252 @@
+# # Force Based Motion Planning
+# ![](circle_swap_pretty.gif) 
+
+# This is an example implementation of the [Force Based Motion Planning (FMP)](https://arxiv.org/pdf/1909.05415.pdf) algorithm. The algorithm is a decentralized motion
+# planning algorithm where individual agents experience attractive forces towards
+# a goal position and repulsive forces from other agents or objects in the environment. 
+# The FMP algorithm is included as a predefined `update_vel` function for use
+# with [`ContinuousSpace`](@ref).
+
+
+# ## Defining the Agent
+
+# First we need to define an `AbstractAgent` struct that has the necessary
+# information for the FMP algorithm to run. `pos`, and `vel` are position and
+# velocity, respectively. `tau` is an agents goal position. `color` is an
+# optional parameter for plotting used in this example. `type` is the agent
+# type: `:A` is an agent, `:O` is an obstacle, and `:T` is a target. `radius`
+# is the radius of the agent. `SSdims` is an optional parameter used for
+# plotting purposes.
+#
+# Agent types have the following distinctions:
+# - :A = agent (has velocity, subject to repulsive forces)
+# - :O = object (has no velocity, cannot move, agents must go around)
+# - :T = target (no velocity, just a projection of agent target zone)
+
+using Agents, Random, Plots, LinearAlgebra, Colors, ProgressMeter
+
+ENV["GKSwstype"]="nul"
+gr()
+cd(@__DIR__)
+
+mutable struct FMP_Agent <: AbstractAgent
+    id::Int
+    pos::NTuple{2, Float64}
+    vel::NTuple{2, Float64}
+    tau::NTuple{2, Float64}
+    color::String
+    type::Symbol
+    radius::Float64
+    SSdims::NTuple{2, Float64}  ## include this for plotting
+    Ni::Array{Tuple{Int64, NTuple{2, Float64}}} ## array of tuples with agent ids and agent positions
+end
+
+# ## Defining the Model
+
+# Next, we define the model. The FMP algorithm has several parameters defined
+# in the original paper which are stored in a struct. Typical values can be
+# generated using the `FMP_Parameter_Init` function and loaded into the model
+# properties. 
+
+# Additionally, note that we pass in `FMP_Update_Vel` as the `update_vel!`
+# keyword argument when we initialize our `ContinuousSpace`.
+
+## define AgentBasedModel (ABM)
+
+function FMP_Model()
+    properties = Dict(:FMP_params=>FMP_Parameter_Init(),
+                      :dt => 0.01,
+                      :num_agents=>100,
+                      :num_steps=>1500,
+                      :step_inc=>2,
+                     )
+
+    #space2d = ContinuousSpace((1,1); periodic = true, update_vel! = FMP_Update_Vel)  #BONE commented this out
```

Comment:
Be sure that the final examples for merging do not have forgotten comments like these